hc/accounts/models.py

```diff
@@ -19,7 +18,7 @@ class Profile(models.Model):
     team_name = models.CharField(max_length=200, blank=True)
     team_access_allowed = models.BooleanField(default=False)
     next_report_date = models.DateTimeField(null=True, blank=True)
-    reports_allowed = models.BooleanField(default=True)
+    reports_allowed = models.CharField(max_length=1, default="0")
```

Comment:
We were avoiding being redundant. By selecting either daily, weekly or monthly reports,  we are already showing an intention to send reports.